src/frequenz/channels/_broadcast.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, name: str, resend_latest: bool = False) -> None:
         Only used for debugging purposes.
         """
 
-        self._resend_latest = resend_latest
+        self.resend_latest = resend_latest
         """Whether to resend the latest value to new receivers."""
```

Comment:
Yes,  I was thinking that it was not relevant in the channel library, because from this point of view, anything is just a message.

But maybe it would be helpful to mention, will do.

Reply:
Thanks!